### ISSUE TITLE
compiler parameters should be configurable

### ISSUE THREAD

**Issue by DavidGamba** [dahu/vim-asciidoc#4]
Hi, thanks for the plugin.

By default, the `asciidoctor` executable will use text for admonition and callout icons. If you define `-a icons=font` or use `:icons: font` in your document, asciidoctor will use foundation icons without any need to define an icons directory. 

Your current compiler configuration is setting different attributes that users might not want to set, for example the `icons` attribute, so I can't get `font` or `text` icons to work.

I would prefer if instead of the `asciidoctor_theme` variable, you could set a `asciidoctor_arguments` variable and send that before the filename. In that way I can set everything I want.

WDYT?

**Comment by dahu** [comment]
You're right. The compiler stuff began as a quick hack. I will think about a better way to handle this; perhaps as you say with an overall compiler_arguments instead of just the few I cater for now.

**Comment by DavidGamba** [comment]
Awesome, thanks!

I forked your repo and right now just removed all args alltogether (I am ok with asciidoctor's defaults when writing a single file). I will see if I come up with something worth sharing. 

Thanks again for a great plugin :-)
